apis/Google.Cloud.Logging.Console/docs/index.md

```diff
@@ -33,18 +33,23 @@ builder.Logging.AddGoogleCloudConsole(options => options.IncludeScopes = true);
 
 ## Log trace correlation
 
-The console formatter supports [log correlation](https://cloud.google.com/logging/docs/view/correlate-logs) for
-[Google Cloud Trace](https://cloud.google.com/trace). This requires a number of conditions to be met:
+The console formatter supports [log correlation](https://cloud.google.com/logging/docs/view/correlate-logs) and
+[Google Cloud Trace log integration](https://cloud.google.com/trace/docs/trace-log-integration) via trace IDs.
+This requires a number of conditions to be met:
 
 - Your application needs to be exporting log entries to Google Cloud Logging via an agent that reads from stdout.
   This happens automatically when running on Google Cloud, and is presumably the reason you're using this
   console formatter.
 - Your application needs to be exporting traces to Google Cloud Trace. This could be performed automatically
   by the runtime, or via explicit code or instrumentation. Google Cloud runtimes export some traces automatically.
+  (For example, Cloud Run will [export traces for all received requests](https://cloud.google.com/run/docs/trace)
```

Comment:
It's not for all received requests, it does sampling. Just removing the "all" should be fine.

Reply:
Great, will do, thanks.